src/catalog/models/df_column.py

```diff
@@ -13,79 +13,124 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 import json
-from enum import Enum
 from typing import List
 
-from sqlalchemy import Column, String, Integer, Boolean
+from sqlalchemy import Column, String, Integer, Boolean, UniqueConstraint
+from sqlalchemy.types import Enum
 
-from src.catalog.database import BaseModel
 from src.catalog.column_type import ColumnType
-from sqlalchemy.types import Enum
+from src.catalog.models.base_model import BaseModel
 
 
 class DataFrameColumn(BaseModel):
     __tablename__ = 'df_column'
 
-    _id = Column('id', Integer, primary_key=True)
     _name = Column('name', String(100))
     _type = Column('type', Enum(ColumnType), default=Enum)
     _is_nullable = Column('is_nullable', Boolean, default=False)
-    _array_dimensions = Column('array_dimensions', String(100), default='[]')
```

Comment:
What if the column type is not ndarray?

Not sure if it's a good idea to store array dimensions as a string.

Reply:
Currently, if it's not ndarray the default values is '[]'

Reply:
sqlalchemy doesn't support mysql arrays so deferring it.